app/scripts/services/http-service.js

```diff
@@ -0,0 +1,125 @@
+'use strict';
+/**
+* @ngdoc service
+* @name angularServicesApp.httpService
+* @description
+* # httpService
+* # a service that abstracts $resource for simple and agnostic CRUD network calls.
+* # any special treatment or data can be abstracted in factories.
+* # will need some work to pass $token and $serverPath for Kronos Schedule.
+* @requires $resource
+*/
+angular.module('angularServicesApp')
+  .service('httpService', function ($resource) {
```

Comment:
The naming standard should be dot notation so we should name this service:

kronos.apps.services.httpService